### ISSUE TITLE
Debugging plug-ins under Big Sur is not easy + ClipIt crash

### ISSUE THREAD

**Issue by 7sharp9** [AuburnSounds/Dplug#574]
Installed via build with:  `sudo dplug-build -c AU --publish`

Looks to be a missing font?

```
Process:               Logic Pro X [51364]
Path:                  /Applications/Logic Pro X.app/Contents/MacOS/Logic Pro X
Identifier:            com.apple.logic10
Version:               10.6.3 (5458)
Build Info:            MALogic-5458000000000000~3 (10U159)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Logic Pro X [51364]
User ID:               501

PlugIn Path:             /Users/USER/*/IPlugEffect.app/Contents/Frameworks/AUv3Framework.framework/Versions/A/AUv3Framework
PlugIn Identifier:       com.AcmeInc.app.IPlugEffect.AUv3Framework
PlugIn Version:          1.0.0 (1.0.0)

Date/Time:             2021-08-18 19:39:56.697 +0100
OS Version:            macOS 11.5.1 (20G80)
Report Version:        12
Bridge OS Version:     5.5 (18P4759a)
Anonymous UUID:        EED5B02E-3F91-E847-FBC4-0EE798A81910

Sleep/Wake UUID:       043CB8F1-480C-441B-A526-F77BE6FAB65E

Time Awake Since Boot: 140000 seconds
Time Since Wake:       16000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
Dublin | 6b34edbe1e2428234532a9a90f0dda622834cd344de3e01f645f6cf5be04bce1 | c226c3f729b55e358699a7f7b18f27a9803c1159 | 2021-07-07_15:09:24
 
Assertion failed: (nvgFindFont(mVG, text.mFont) != -1 && "No font found - did you forget to load it?"), function PrepareAndMeasureText, file ../../../IGraphics/Drawing/IGraphicsNanoVG.cpp, line 624.
```

**Comment by p0nce** [comment]
Hello,
- I'm not sure if `--publish` is actually implemented, I think not
- Pretty sure this is an Iplug plugin crashing, there is no `nvgFindFont` in Dplug 
=> https://github.com/iPlug2

is there any other log in Console.app?

**Comment by 7sharp9** [comment]
Ooops, I was testing plug the other data and must have mistook the name in Logic, the Clip it plugin simply does not show up in Logic.

```
validating Audio Unit CLIP It by Witty Audio:

    AU Validation Tool
    Version: 1.8.0 
    Copyright 2003-2019, Apple Inc. All Rights Reserved.
    Specify -h (-help) for command options

--------------------------------------------------
VALIDATING AUDIO UNIT: 'aumf' - 'ClPi' - 'Wity'
--------------------------------------------------
Manufacturer String: Witty Audio
AudioUnit Name: CLIP It
Component Version: 1.0.0 (0x10000)

* * PASS
--------------------------------------------------
TESTING OPEN TIMES:
COLD:
FATAL ERROR: OpenAComponent: result: -1,0xFFFFFFFF


validation result: couldn’t be opened
```

**Comment by p0nce** [comment]
Yeah this is unfortunate on Big Sur. 
Plugins must be codesigned/notarized or every host will fail to open them.

It's not really a Dplug thing.
We should find a way to enable debugging under Big Sur, but you can do it with notarization (and this requires the Apple Developer account, codesign cert setup, etc...)

**Comment by 7sharp9** [comment]
publish definitely  works for the other example `distort`, the issue seems to be in  Clipit

**Comment by p0nce** [comment]
If you need a workaround right now because you can't develop on an other machine:
https://github.com/AuburnSounds/Dplug/wiki/Dplug-Installer-Guide
https://github.com/AuburnSounds/Dplug/wiki/Notarization-Guide

**Comment by 7sharp9** [comment]
I was curious as to why the the distort example works but the clip one doesn't?

**Comment by p0nce** [comment]
This is strange, I have no idea right now.

**Comment by p0nce** [comment]
This is fixed if you upgrade Big Sur to 11.5.2
For your AU problem, it's because new AU don't show up in Big Sur until rebooting.
--publish will now use sudo by itself in ~master
